iamproperty/iamproperty.github.io
# Ds 115/charts

## Summary of Changes:
1. Charts CSS
2. Charts Vue component
3. Charts docs and examples
4. placeholder for article to be written next

## Rationale
Have a way to represent chart type data

## How to Test
Open up the preview site (see the below netlify comment below) and navigate to /components/charts

## Ticket(s)
DS-115

## Checklist

## Notes
